code/datums/elements/_element.dm

```diff
@@ -60,7 +60,7 @@
 	var/datum/element/ele = SSdcs.GetElement(arguments)
 	arguments[1] = src
 	if(ele.Attach(arglist(arguments)) == ELEMENT_INCOMPATIBLE)
-		CRASH("Incompatible [arguments[1]] assigned to a [type]! args: [json_encode(args)]")
+		CRASH("Incompatible [ele] assigned to a [type]! args: [json_encode(args)]")
```

Comment:
oh so THAT's why this runtime was so weird

Reply:
yeaaaaah it's funny. hopefully we can actually figure what the hell is actually going on now lol